### DISCUSSION TITLE
Compile Error when installing a new project with Nextjs

### DISCUSSION THREAD

**Discussion by Favourz1** [vercel/next.js#56202] [Help]
### Summary

I started a simple nextjs application but it fails to compile when i start the development server. I get the error

```javascript
../../../../node_modules/buffer/index.js
Module parse failed: Unexpected token (1119:5)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|   Object.setPrototypeOf(newBuf, Buffer.prototype)
| 
>   ret
``` 

I deleted my node modules, cleared cache and reinstalled yet it still doesn't work. How can i fix this?

### Additional information

```js
../../../../node_modules/buffer/index.js
Module parse failed: Unexpected token (1119:5)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|   Object.setPrototypeOf(newBuf, Buffer.prototype)
| 
>   ret
```

### Example

npx create-next-app  on my  machine gives the error.

**Comment by icyJoseph** [comment]
What's your Node.js version?

**Reply by Favourz1** [reply]
Node 16.16 @icyJoseph 

Tried it with Node 18.5 still didn't work

**Reply by icyJoseph** [reply]
Mm that's quite weird.

What OS do you have? `npx next info` can output some hints.

**Reply by Favourz1** [reply]
![image](https://github.com/vercel/next.js/assets/87978245/a920d6de-7c9a-4e56-959d-6c034044d10b)
This is what i got.

**Reply by Favourz1** [reply]
Any ideas @icyJoseph ?

**Reply by icyJoseph** [reply]
Not really mate. You just run create-next-app, and tried to start the server, and that's it right? It errors out.

**Reply by Favourz1** [reply]
Yes that's what i did and got the error.

Do you think it would be a problem with settings in my machine? As i have installed and used next severally.

Because the error points about webpack loader configuration.

**Reply by icyJoseph** [reply]
Yeah that's the strange part.

Maybe double check that webpack nor next.js are globally installed. Something like `npm list -g` though I don't think that is it.

It's like, unexpected token should happen when the interpreter cannot parse the code, or just straight up badly written code, but then everyone would be getting the error though.

**Reply by icyJoseph** [reply]
Btw, what do you get if you run:

```console
node -p "process.arch"
```

Does it print a matching version to your OS?
